Invalid comment source generation in WCT

## Current behavior

The compilation fails with the following errors (formatting breaks because of backquotes):

D:\a\1\s\Microsoft.Toolkit.Uwp.UI.Controls.Layout\obj\Release\xamarinmac20\g\NativeCtorsGenerator\Microsoft_Toolkit_Uwp_UI_Controls_PropertyChangeEventSource_TPropertyType__fca3e7386d803e365f0d121b8533af45.g.cs(18,52): error CS1584: XML comment has syntactically incorrect cref attribute 'Microsoft.Toolkit.Uwp.UI.Controls.PropertyChangeEventSource`1' [D:\a\1\s\Microsoft.Toolkit.Uwp.UI.Controls.Layout\Microsoft.Toolkit.Uwp.UI.Controls.Layout.csproj]
         D:\a\1\s\Microsoft.Toolkit.Uwp.UI.Controls.Layout\obj\Release\xamarinmac20\g\NativeCtorsGenerator\Microsoft_Toolkit_Uwp_UI_Controls_PropertyChangeEventSource_TPropertyType__fca3e7386d803e365f0d121b8533af45.g.cs(18,111): error CS1658: Unexpected character 'backquote'. See also error CS1056. [D:\a\1\s\Microsoft.Toolkit.Uwp.UI.Controls.Layout\Microsoft.Toolkit.Uwp.UI.Controls.Layout.csproj]
         D:\a\1\s\Microsoft.Toolkit.Uwp.UI.Controls.Layout\obj\Release\xamarinmac20\g\NativeCtorsGenerator\Microsoft_Toolkit_Uwp_UI_Controls_PropertyChangeEventSource_TPropertyType__fca3e7386d803e365f0d121b8533af45.g.cs(18,112): error CS1658: Unexpected character '1'. See also error CS1056. [D:\a\1\s\Microsoft.Toolkit.Uwp.UI.Controls.Layout\Microsoft.Toolkit.Uwp.UI.Controls.Layout.csproj]

The generated code is as follows:

```csharp
#if True
		/// <summary>
		/// Initializes a new instance of the <see cref="Microsoft.Toolkit.Uwp.UI.Controls.PropertyChangeEventSource`1"/> class.
		/// </summary>
		public PropertyChangeEventSource() { }
#endif
```

## Expected behavior

The compilation does not fail.

--- a/src/SourceGenerators/Uno.UI.SourceGenerators/NativeCtor/NativeCtorsGenerator.cs
+++ b/src/SourceGenerators/Uno.UI.SourceGenerators/NativeCtor/NativeCtorsGenerator.cs
@@ -59,7 +59,7 @@ namespace {0}
 	{{
 #if {2}
 		/// <summary>
-		/// Initializes a new instance of the <see cref=""{4}""/> class.
+		/// Initializes a new instance of the class.
 		/// </summary>
 		public {3}() {{ }}
 #endif
@@ -154,8 +154,7 @@ namespace {0}
 								typeSymbol.ContainingNamespace,
 								smallSymbolName,
 								NeedsExplicitDefaultCtor(typeSymbol),
-								SyntaxFacts.GetKeywordKind(typeSymbol.Name) == SyntaxKind.None ? typeSymbol.Name : "@" + typeSymbol.Name,
-								DocumentationCommentId.CreateReferenceId(typeSymbol)
+								SyntaxFacts.GetKeywordKind(typeSymbol.Name) == SyntaxKind.None ? typeSymbol.Name : "@" + typeSymbol.Name
 							)
 						);
 					}
@@ -180,8 +179,7 @@ namespace {0}
 								typeSymbol.ContainingNamespace,
 								smallSymbolName,
 								NeedsExplicitDefaultCtor(typeSymbol),
-								SyntaxFacts.GetKeywordKind(typeSymbol.Name) == SyntaxKind.None ? typeSymbol.Name : "@" + typeSymbol.Name,
-								DocumentationCommentId.CreateReferenceId(typeSymbol)
+								SyntaxFacts.GetKeywordKind(typeSymbol.Name) == SyntaxKind.None ? typeSymbol.Name : "@" + typeSymbol.Name
 							)
 						);
 					}

